src/testing/jest/test/jest-config.spec.ts

```diff
@@ -19,6 +19,17 @@ describe('jest-config', () => {
     expect(jestArgv.maxWorkers).toBe(2);
   });
 
+  it('should pass --outputFile=path/to/file', () => {
```

Comment:
Minor, it's not immediately clear to me what this is testing from this description. Can we work the expected result into the test name here? Maybe something like:

```ts
  it("marks 'outputFile' as a jest argument", () => {
```